### ISSUE TITLE
[UX] Layout listing: "No conditions set." doesn't make sense for the 2 default layouts

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4873]
Both of these layouts have `No conditions set.` in their "Visibility conditions" column:

![image](https://user-images.githubusercontent.com/2423362/104431960-2ee04280-5591-11eb-8ecf-30ad0cca78d7.png)

I find that not to be entirely true, as in it simply doesn't make sense from an end user point of view.

As a new user, looking at this page with fresh eyes, I do notice that the name of one of these layouts includes the word "administrative" in it, but then looking at the conditions column, the `No conditions set.` text buffles me. It makes me wonder how one of these two layouts is being selected then.

I thought that adding a pseudo-condition to each of the default layouts would make more sense:

![image](https://user-images.githubusercontent.com/2423362/104459834-f868ee80-55b5-11eb-9a11-6ca459e18772.png)

...and we could even move the "used on all paths without a specific layout" text in the "Visibility conditions" column. Like so:

![image](https://user-images.githubusercontent.com/2423362/104459762-d7a09900-55b5-11eb-874a-6969b0eb354f.png)

**Comment by klonos** [comment]
PoC PR: https://github.com/backdrop/backdrop/pull/3493

**Comment by jenlampton** [comment]
Hm, I don't know about this....

Love it 👍 
* There are no conditions on the Default layout, so I think stating `No conditions set` is useful here. 
* The administrative layout **does** have the condition `is an administrative page`, I like this addition.

Needs work 👎 
* The Default layout **does not** have the condition `is not an administrative page`. I think indicating that it does in the interface is misleading. 
* Every layout has the condition `No other layout matches`, but if we were to add that text to all layouts, it won't add any value. 


The reason the default layout seems like it has the condition `is not an administrative page` is because it did NOT match the admin layout condition. This is exactly the same as why it's not a node page, why it's not a user page, why it's not a term page, and why it's not any layout with a defined path, but I don't think we should add those explanations to the default layout either...

**Comment by stpaultim** [comment]
> Every layout has the condition No other layout matches, but if we were to add that text to all layouts, it won't add any value.

This isn't true is it? The every layout has the condition? One layout might match for being a node while another layouts matches for both being a node and being node/34. 

I would think that "No other layouts matches" is exclusive to the default. I think it is reasonable to say that the default layout is applied if "no other layout matches," but for almost any other layout there is at least the chance that some other layout matches, but has not been applied based upon some prioritization scheme which I am still trying to understand. :-)

**Comment by jenlampton** [comment]
> This isn't true is it? That every layout has the condition "No other layout matches"?

You're right. If there are two layouts in a group (like `node/%`) that match, the first match would be selected. 

> "No other layouts matches" is a requirement for the default layout, by definition.

yes, that's why it's called the "Default" layout. It will be used by default, when no other layout is in use. Ah...  I see what you are getting at with "No other layout matches". Perhaps a rephrasing would help me here.

Is the intent to write something that's specific to only the Default layout? Or something that's specific to both Default layout and the Default admin layout. Are we trying to define "Default"?

**Comment by klonos** [comment]
> Ah... I see what you are getting at with "No other layout matches". Perhaps a rephrasing would help me here.

Yeah, that was an attempt at a quick mockup in the form of a PR that people can play with. That "pseudo-condition" should be rephrased. Suggestions?

**Comment by philsward** [comment]
"Used if no other layouts match"
